src/pineko/evolve.py

```diff
@@ -1,21 +1,36 @@
 """Tools related to evolution/eko."""
 import copy
+import os
 import pathlib
 
+import eko
 import eko.basis_rotation as br
+import eko.compatibility
 import numpy as np
 import pineappl
 import rich
 import rich.box
 import rich.panel
 import yaml
 
-from . import check, comparator, version
+from . import check, comparator, ekompatibility, version
 
+DEFAULT_PIDS = [-5, -4, -3, -2, -1, 21, 22, 1, 2, 3, 4, 5]
 
-def write_operator_card_from_file(pineappl_path, default_card_path, card_path, xif):
+
+def write_operator_card_from_file(
+    pineappl_path: os.PathLike,
+    default_card_path: os.PathLike,
+    card_path: os.PathLike,
+    xif: float,
+    tcard_path: os.PathLike,
```

Comment:
Not really relevant but for me it would be more sensible to have `tcard_path` together with the other paths (i.e. before `xif`).

Reply:
I considered it as well, but in the end is something temporary, and we might want to drop it at some point (together with all the compatibility stuffs).

It is not there to stay, and at the moment I'd prioritize merging and keep going. We will reconsider in any case ^^